content/docs/06-integrations/00-antrea-cni.md

```diff
@@ -0,0 +1,106 @@
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Antrea CNI
+
+Palette supports Antrea controller network interface (CNI) for Kubernetes clusters. Antrea  CNI enables each pod to have exclusive IP addresses from the subnet with direct accessibility. 
+
+Antrea leverages [Open vSwitch](https://www.openvswitch.org/) to implement pod networking and security features. Open vSwitch enables Antrea to implement Kubernetes network policies efficiently.
+
+# Supported Versions
+
+**1.9.x**
+
+# Prerequisites
+
+- Enable the ``enableNodeIPAM`` integrated NodeIPAM controller within the Antrea controller.
+- When deploying a cluster using kubeadm, specify the ``--pod-network-cidr <cidr>`` option and provide the IP address with the classless inter-domain routing (CIDR). For example: 
+
+    ``--pod-network-cidr=10.244.0.0/16``
+
+    <br />
+
+    <WarningBox>
+
+    To avoid overlapping your pod network with any of your host networks, you should think of a suitable CIDR block to specify if you deploy a cluster using ``kubeadm init`` with ``--pod-network-cidr <cidr>`` or as a replacement in your network plugin's YAML.
+
+    </WarningBox>
+
+- The Open vSwitch kernel module must be present on every Kubernetes node.
+
+# Parameters
+
+The Antrea CNI pack supports the following parameters.
+
+| Parameter | Description | Required (Y/N) |
+|-----------|-------------|---------|
+| nodeIPAM:enable | This enables the integrated NodeIPAM controller within the Antrea controller. The default is `false`. | Y |
+| clusterCIDRs | CIDR ranges for pods in the cluster. The CIDRs could be either IPv4 or IPv6. You can specify up to one CIDR for each IP family. | N |
+| serviceCIDRv6 | IPv6 CIDR ranges reserved for Services. | N |
+| nodeCIDRMaskSizeIPv4 | Mask size for IPv4 Node CIDR in IPv4 or dual-stack cluster. | N |
+| nodeCIDRMaskSizeIPv6 | Mask size for IPv6 Node CIDR in IPv6 or dual-stack cluster. | N |
+| NodeIPAM | The feature toggle for ``antrea-controller``. The default is `false`. If you use CIDR ranges, set this to ``true``.  | N |
+| ServiceExternalIP | The feature toggle for ``antrea-agent`` and ``antrea-controller``. If you use the LoadBalancer service, set this to ``true``. | N |
+
+
+# Usage
+
+Kubernetes network policies are supported by default.
+
+Antrea supports LoadBalancer services. Typically, implementing LoadBalancer services requires an external load balancer that is implemented by the Kubernetes cloud provider. 
+
+Antrea provides two options for supporting LoadBalancer services without using an external load balancer:
+- Using Antrea’s built-in external IP management for Services of type LoadBalancer.
+
+
+- Leveraging MetalLB.
+
+For detailed information, refer to Antrea’s [Service of type LoadBalancer](https://antrea.io/docs/v1.9.0/docs/service-loadbalancer) documentation. 
+
+# Troubleshooting
+
+Ensure you have provided a non-overlapping IP address for your pod network  in the Kubernetes pack. 
```

Comment:
IP address provided in k8s layer will always take preference. Refer to my previous comment